conbench/templates/app.html

```diff
@@ -196,7 +196,7 @@ <h4 class="modal-title"><span class="glyphicon glyphicon-trash submenu"></span>&
         table.columns(column_indexes).every(function() {
             var that = this;
             var title = this.header().textContent;
-            $(`<input class="custom-input-text custom-search-box" type="text" placeholder="🔍 ${title}"
+            $(`<br /><input class="custom-input-text custom-search-box" type="text" placeholder="🔍 filter"
```

Comment:
Note: the additional break here is so that the input field is guaranteed to go below the column heading.

Reply:
😂 This feels like one of those "Design is my passion ❤️ CSS" memes right here. 

In seriousness: this is totally fine and I like that the sorting is next to the inputboxes when they are there